solvers/python_pgd.py

```diff
@@ -10,15 +10,15 @@ class Solver(BaseSolver):
     stopping_strategy = "callback"
 
     # any parameter defined here is accessible as a class attribute
-    parameters = {'use_acceleration': [False, True]}
+    parameters = {"use_acceleration": [False, True]}
```

Comment:
These changes are also unrelated

Reply:
I have removed all but the unique file intended here, should solve the problem